src/rest/types.hpp

```diff
@@ -60,6 +66,7 @@ enum class HttpStatusCode : std::uint16_t
 {
     kStatusOk                  = 200,
     kStatusAccepted            = 202,
+    kNoContent                 = 204,
     kStatusBadRequest          = 400,
     kStatusResourceNotFound    = 404,
     kStatusMethodNotAllowed    = 405,
```

Comment:
I am happy if you remove all "Status" from those enum constants :)

Reply:
Not against it, but I would prefer to do it in a follow up as it leads to code changes throughout `resource.cpp`.

Reply:
SGTM, thanks!